lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -991,7 +991,10 @@ inline bool hasDbgInstParentScopeIdx(
     ParentScopeIdx = TypeMember::OpenCL::ParentIdx;
     return true;
   case SPIRVDebug::TypeInheritance:
-    ParentScopeIdx = TypeInheritance::ParentIdx;
+    if (ExtKind == SPIRV::SPIRVEIS_OpenCL_DebugInfo_100)
```

Comment:
Lets separate indexes for `TypeInheritance` in OpenCL and NonSemantic namespaces (like it's done for other instructions).

Reply:
Done.